insights/config/specs.py

```diff
@@ -22,7 +22,8 @@
     "brctl_show"                : CommandSpec("/usr/sbin/brctl show"),
     "candlepin.log"             : First([SimpleFileSpec(r"var/log/candlepin/candlepin.log", large_content=True),
                                     SimpleFileSpec(r"sos_commands/foreman/foreman-debug/var/log/candlepin/candlepin.log", large_content=True)]),
-    "candlepin_error_log"       : NoneGroup([SimpleFileSpec(r"sos_commands/foreman/foreman-debug/var/log/candlepin/error.log", large_content=True)]),
+    "candlepin_error_log"       : First([SimpleFileSpec(r"var/log/candlepin/error.log", large_content=True),
+                                    SimpleFileSpec(r"sos_commands/foreman/foreman-debug/var/log/candlepin/error.log", large_content=True)]),
```

Comment:
@sagaraivale I believe these specs should still be wrapped in a `NoneGroup` since they are targeted for archives only.

Reply:
@xiangce thanks for clearing that up.